src/pages/api-reference/layers/vector-layer.md

```diff
@@ -50,4 +61,8 @@ Whether you've made a small tweak to one of Esri's hosted vector basemaps or pub
 ```js
 var map = L.map('map').setView([37.75,-122.45], 12);
 L.esri.Vector.layer('bd505ce3efff479bb4e87b182f180159').addTo(map);
+
+// Item page for Light Gray Canvas Reference (Local Language) is https://www.arcgis.com/home/item.html?id=3ffec1551cd14606a286622c634b0bb4
+// You can pass in the item id to add this vector basemap
+L.esri.Vector.layer('3ffec1551cd14606a286622c634b0bb4').addTo(map);
```

Comment:
it'd be better to share a 'self contained' basemap.

Reply:
So, would it be better to share the link to the "map" items within the [basemaps group](https://www.arcgis.com/home/group.html?id=30de8da907d240a0bccd5ad3ff25ef4a&view=list&start=1&focus=maps#content)?

And then also to reference a web map item id in the example?

Reply:
I think you meant a basemap that contains features and labels.  Had a hell of a day trying to rebuild a Drupal development site.

Reply:
sorry, i mean vector tile item that already has labels (streets, topo, nav, etc.)